mmaction/core/evaluation/ava_evaluation/per_image_evaluation.py

```diff
@@ -0,0 +1,449 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Agree, but I think the priority now is to support the new feature. We can polish the code in the next version. Since the code is copied from the official ActivityNet repo, it is bug free and can be used without any further changes.